packages/plugin-ext/src/common/plugin-api-rpc.ts

```diff
@@ -1255,6 +1255,7 @@ export interface TaskDto {
     // Provide a more specific type when necessary (see ProblemMatcherContribution)
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     problemMatcher?: any;
+    group?: string;
```

Comment:
Should the `TaskDto` support all the options available in the task schema (i.e. strings or `{kind: string}`)? Or is there reason to believe that we won't receive data in that form?

Reply:
hmm!, it's a good question but I am not sure about that, it goes outside the scope of the current fix